src/App/Link.cpp

```diff
@@ -902,6 +1355,10 @@ void LinkBaseExtension::onExtendedUnsetupObject() {
     getElementListProperty()->setValue();
     for(auto obj : objs)
         detachElement(obj);
+    if (auto obj = getLinkCopyOnChangeGroupValue()) {
+        if(obj->getNameInDocument() && !obj->isRemoving())
```

Comment:
I got this pattern from older code like `App::Origin` to auto remove it's child when it is removed. `isRemoving()` test against a status bit set by `App::Document` while removing the object to prevent error message. I guess it is no longer needed, as the property link holding the children will be cleared when they are removed. Not entirely sure.